src/pages/editor/PublishStrategySettings.tsx

```diff
@@ -22,12 +22,12 @@ const StyledItemContainer = styled('div')({
   alignItems: 'center',
 });
 
-const StyledAccessRight = styled('div')({
+export const StyledAccessRight = styled('div')({
   display: 'flex',
   gap: '0.5rem',
 });
 
-const StyledAccessRightsContainer = styled('div')({
+export const StyledAccessRightsContainer = styled('div')({
```

Comment:
detaljer, men om disse skal brukes tror jeg det kan lønne seg å flytte dem til `PublishingStrategyOverview`.
Dette fordi Overview skal vises for alle som kan se Settings, mens det samme ikke gjelder motsatt. Ikke helt sikker på dette altså, men jeg tror det vil hjelpe ved lazy-loading om vi gjør det motsatt, ellers vil denne fila som egentlig bare er relevant for redaktører, også lastet for alle ikke-redaktører